docs/templates/package.json

```diff
@@ -54,7 +55,7 @@
     "ts-node": "5.0.1",
     "typescript": "^2.8.1"
   },
-  "dependencies": {
+  "peerDependencies": {
     "@aerogear/core": "0.1.0"
```

Comment:
I think `dependencies` is probably right? Do we need it in `devDependencies` as well?